index.js

```diff
@@ -44,6 +44,14 @@ var isArray = Array.isArray;
 // interdependence between different functions.
 var conjure;
 
+// a special hasOwnProperty function so we can override the built-in one
+
+var hasOwnProperty = Object.prototype.hasOwnProperty;
+
+function hop(obj, propName) {
```

Comment:
Extending `Object.prototype` is not an option as it would affect the entire process. We only want it overridden on Tomes (array and object tomes that is), and we don't want to invoke it in the internals (since there we do care about those properties that are hidden from view).